README.md

```diff
@@ -77,14 +81,12 @@ We have an open source quick start template for nodejs.
  - `tcurl -p localhost:9000 my-service MyService::health_v1 -t ./thrift/service.thrift`
  - `tcurl -p 127.0.0.1:21300 my-service MyService::health_v1 -t ./thrift/service.thrift`
 
-What this does is start a hello world tchannel app in nodejs that
-registers with hyperbahn.
-
 You can `tcurl` it directly at `localhost:9000` or `tcurl` it through
-hyperbahn at `127.0.0.1:21300`.
+Hyperbahn at `127.0.0.1:21300`. If you're unfamiliar with `tcurl`, the
+command-line tool to call TChannel servers, learn about it [here][tcurl].
 
-Because the service is available on Hyperbahn anyone can talk to it
-if they can talk to Hyperbahn.
+Once a service is available on Hyperbahn, any other service on Hyperbahn
```

Comment:
Am I correct in thinking any other service on THAT Hyperbahn cluster can talk to it?
It's not like some little startup can set up a Hyperbahn cluster, put Service A on it, and then have Service A talk to services at Uber that are on our Hyperbahn network, correct?